app/controllers/api/verify/password_reset_controller.rb

```diff
@@ -6,7 +6,7 @@ class PasswordResetController < BaseController
       def create
         analytics.idv_forgot_password_confirmed
         request_id = sp_session[:request_id]
-        email = current_user.email
+        email = current_user.confirmed_email_addresses.first.email
         reset_password(email, request_id)
```

Comment:
should this go to all email addresses maybe? or to just the email that the user typed in?

Reply:
In both IDV reset password controllers the user doesn't type in an email address:

![image](https://user-images.githubusercontent.com/1430443/172714988-1f742ef2-386e-4d48-9aaf-e872d6f43ab6.png)

We could send to all emails but it'd also need a content update similar to https://github.com/18F/identity-idp/pull/6474#discussion_r892807944 since this also shows the email it was sent to:

![image](https://user-images.githubusercontent.com/1430443/172715797-d3a8d87a-6fe1-4178-9d8e-e849cdfaca1a.png)

Reply:
ah thanks for the screenshots, that clarifies a ton

Reply:
WDYT of filing a ticket to ask UX to look in to sending to all emails vs one email? not blocking for this PR

Reply:
Sounds good to me! I created [LG-6597](https://cm-jira.usa.gov/browse/LG-6597)